lib/stdlib/src/features.erl

```diff
@@ -0,0 +1,308 @@
+%%
+%% %CopyrightBegin%
+%%
+%% Copyright Ericsson AB 2021-2022. All Rights Reserved.
+%%
+%% Licensed under the Apache License, Version 2.0 (the "License");
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%%
+%%     http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+%%
+%% %CopyrightEnd%
+%%
+-module(features).
```

Comment:
`features` is a fairly "common" word and it might easily clash with existing code. Additionally this module is not "crucial" and I'd expect it won't be used much in user code. Could this follow naming convention similar to other compiler-related modules with `erl_features` - following `erl_parse`, `erl_scan`, and friends.

Reply:
Good point.  We'll discuss a better, more "internal" name for the module.

Reply:
I agree.

IMHO, from shell experience perspective but also current EEP description `erl_features:F` is nicer than `features:F` because EEP is specific that it deals with "Erlang language features"